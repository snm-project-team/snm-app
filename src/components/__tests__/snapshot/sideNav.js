import React from 'react';
import renderer from 'react-test-renderer';
import jest from 'jest-mock';
import MainMenu from '../../sideMenu/mainMenu';

const mockFn = jest.fn();

it('MainMenu画面のスナップショットテスト', () => {
  const rendered = renderer.create(
    <MainMenu
      signOut={mockFn}
    />,
  ).toJSON();
  expect(rendered).toMatchSnapshot();
});
