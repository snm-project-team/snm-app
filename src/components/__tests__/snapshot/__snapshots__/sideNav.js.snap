// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenu画面のスナップショットテスト 1`] = `
<View
  closeOnRowBeginSwipe={false}
  closeOnRowPress={true}
  closeOnScroll={true}
  directionalDistanceChangeThreshold={2}
  disableLeftSwipe={false}
  disableRightSwipe={false}
  leftOpenValue={0}
  previewFirstRow={false}
  recalculateHiddenLayout={false}
  rightOpenValue={0}
  style={Object {}}
  swipeToOpenPercent={50}
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={null}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "transparent",
          "borderBottomWidth": 0.5,
          "borderColor": "#c9c9c9",
          "flexDirection": "row",
          "marginLeft": 16,
          "paddingBottom": 13,
          "paddingRight": 16,
          "paddingTop": 13,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignSelf": "center",
            "color": "#000",
            "fontFamily": "System",
            "fontSize": 16,
          }
        }
        uppercase={false}
      >
        グループ管理
      </Text>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={null}
  >
    <View
      id="signOut"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "transparent",
          "borderBottomWidth": 0.5,
          "borderColor": "#c9c9c9",
          "flexDirection": "row",
          "marginLeft": 16,
          "paddingBottom": 13,
          "paddingRight": 16,
          "paddingTop": 13,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignSelf": "center",
            "color": "#000",
            "fontFamily": "System",
            "fontSize": 16,
          }
        }
        uppercase={false}
      >
        サインアウト
      </Text>
    </View>
  </View>
</View>
`;
